### DISCUSSION TITLE
separate static config data from working data and temp files

### DISCUSSION THREAD

**Discussion by mknj** [dani-garcia/vaultwarden#2339] [Ideas]
For security and management reasons i separate static(read-only) configuration data, working data and temp files.

The vaultwarden /data directory contains files of all three categories. This complicates mounting them to separate volumes.

Being able to set different directories for icon_cache, sqlite and rsa_key via environment variables or the config.json would solve my issue.

**Comment by mknj** [comment]
sorry, just found https://github.com/dani-garcia/vaultwarden/wiki/Changing-persistent-data-location. How to delete a discussion?

**Comment by BlackDex** [comment]
I sugges to leave it here, since other people may have the same question and then can few yours :).
